tools/minHash/.shed.yml

```diff
@@ -0,0 +1,17 @@
+name: minHash
+owner: joachimwolff
+description: A tool to compute the nearest neighbors in a high dimensional sparse dataset.
+
+long_description: |
+  MinHash is an approximate nearest neighbors search algorithm for high dimensional sparse datasets. It computes first a candidate set with MinHash and after the 
+  exact neighbors with the euclidean distance or cosine similarity.
+
+categories:
+  - Machine Learning
+  
+remote_repository_url: https://github.com/joachimwolff/galaxytools/tree/minHash/tools/minHash
+
+auto_tool_repositories:
```

Comment:
This is only needed if you have more than 1 tool in a repo that you want to demultiplex.